### ISSUE TITLE
Reassociation does not work properly

### ISSUE THREAD

**Issue by frozenspider** [moretension/duti#56]
In the past, I used duti v1.5.4_1 (in the form of `duti -v input-file.duti` if that's of any significance) to associate a bunch of text format with VS code (UTI `com.microsoft.VSCode`). As far as I remember, that worked out fine.

However, later on macOS Sonoma 14.2.1 I decided that VS code is too resource-hungry to serve as a general-purpose text editor and switched to Sublime Text (UTI `com.sublimetext.4`). I tried to reassociate text formats with it (using the same `duti -v`) and this lead to a weird situation.

When right-clicking text files in Finder, it shows `Open With > Sublime Text.app (default)`.
<img width="501" alt="image" src="https://github.com/moretension/duti/assets/2077017/b3cc08ef-1708-49da-a70b-244a8250946d">

`Get Info > Open With` shows pretty much the same:
<img width="401" alt="image" src="https://github.com/moretension/duti/assets/2077017/483636f7-1ca1-4743-8249-335f095eadb9">

However, when double-clicking a file, it opens in VS code!

If I change an app in `Get Info > Open With` to something else and then back to Sublime Text, and then click `Change All`, files of this extension will now properly open in Sublime Text upon double click. However, this defeats the purpose of using `duti` to begin with.

**Comment by mxcl** [comment]
Yeah I can reproduce, `duti` no longer seems to work while before the latest macOS major it did. No clue why.

**Comment by resistor4u** [comment]
Is it a permissions/sandbox thing?
